site/src/pages/components/enhanced-range-input.explainer.mdx

```diff
@@ -0,0 +1,479 @@
+
+### WebKit/Blink (Chrome/Opera/Safari):
+
+```css
+::-webkit-slider-thumb { /* Styles the thumb */ }
+::-webkit-slider-runnable-track { /* Styles the track */ }
+```
+
+### Firefox:
+
+```css
+::-moz-range-thumb { /* Styles the thumb */ }
+::-moz-range-track { /* Styles the track */ }
+::-moz-range-progress { /* Styles the progress/fill below the thumb */ }
+```
+
+### IE/Edge:
+
+```css
+::-ms-thumb { /* Styles the thumb */ }
+::-ms-track { /* Styles the track */ }
+::-ms-fill-lower { /* Styles the progress/fill below the thumb */ }
+::-ms-fill-upper { /* Styles the fill above the thumb */ }
+```
+
+This fragmentation makes it difficult for developers to create consistent range input styles across browsers, often requiring browser-specific code or fallbacks.
+
+## Proposed Solution
+
+### Standardized Anatomy
+
+We propose standardizing the internal structure of range inputs across all browsers. This will ensure consistent styling and behavior, regardless of the browser implementation.
+
+<RangeAnatomy/>
+
+### General Structure
+
+The standardized anatomy includes:
+
+- `::range-track`: The main track along which the thumb(s) move.
+- `::range-fill`: The filled portion of the track, typically between the minimum value and the thumb (or between thumbs for multi-handle ranges).
+- `::range-thumb`: The draggable handle(s) used to select values.
+- `::range-segment`: Sections of the track between handles in multi-handle ranges.
+- `::range-tick`: Optional tick marks along the track for value representation. (when datalist is paired.)
+- `::range-tick-label`: Labels associated with tick marks. (when datalist is paired.)
+
+### Styling API
+
+To enable comprehensive styling of range inputs, we propose introducing a new CSS `appearance` value and a set of pseudo-elements:
+
+```css
+input[type="range"] {
+    appearance: base-range;
+}
+```
+
+This opt-in approach allows developers to access enhanced styling capabilities while maintaining backwards compatibility.
+
+### Multi-Handle Support
+
+We propose extending the range input to support multiple handles, starting with dual-handle support as the most common use case. This will be achieved through new HTML attributes and corresponding JavaScript APIs.
+
+### Accessibility Enhancements
+
+To improve accessibility, especially for multi-handle ranges, we propose making individual thumbs focusable and providing clear audio cues for screen readers.
+
+### Progressive Enhancement
+
+The reason for opting in is to ensure [backward compatibility](https://chromestatus.com/metrics/feature/timeline/popularity/34) with existing range inputs. Since current implementations are a mess, it feels like a safer bet. Browsers that do not support the new features will fall back to the current behavior of range inputs, and it opens up the window of targeting support via CSS (@supports).
```

Comment:
I personally would prefer to see a new element than placing **all** of the parts within a psueudo elements. I think there is value in landing the pseudo elements as well as landing a new element as there will be usecases that won't be achievable by pseudo elements along (this will apply checkbox, radio, password, color, file, and many others).